libs/application/templates/financial-statements-inao/src/fields/OperatingYear/index.tsx

```diff
@@ -5,22 +5,41 @@ import {
   ContentBlock,
   SkeletonLoader,
 } from '@island.is/island-ui/core'
+import { Application } from '@island.is/application/types'
 import { useLocale } from '@island.is/localization'
 import { m } from '../../lib/messages'
-import { ABOUTIDS, BACKYEARSLIMITFALLBACK } from '../../lib/constants'
+import {
+  ABOUTIDS,
+  BACKYEARSLIMITFALLBACK,
```

Comment:
'BACKYEARSLIMITFALLBACK' is defined but never used. Allowed unused vars must match /^_[a-zA-Z0-9]*$/u